setup.py

```diff
@@ -1,23 +0,0 @@
-from typing import Sequence
-
-from setuptools import find_packages, setup
-
-VERSION = "25.9.0.dev0"
-
-
-def get_requirements() -> Sequence[str]:
```

Comment:
afaict sdists and wheels are not being built for snuba so deleting this altogether